.bazelrc

```diff
@@ -27,6 +27,13 @@ build --fission=dbg
 # upto-date
 build:release --define manual_stamp=manual_stamp
 
+# release_debug - an optimized release build containing the debug information
+build:release_debug -c opt
```

Comment:
should we pass same flags as Envoy optimized build does? https://github.com/envoyproxy/envoy/blob/main/bazel/README.md?plain=1#L574